### ISSUE TITLE
A mistake about computing bias term

### ISSUE THREAD

**Issue by Jun-jie-Huang** [ajtulloch/svmpy#6]
Hi, I've implemented a soft-margin SVM with kernel function and I referred to yours. I found there's a problem in your code to compute the bias term in the training process in this [line.](https://github.com/ajtulloch/svmpy/blob/ff7b138be6bd1657666f0919f8b6271bfc6df9c1/svmpy/svm.py#L42). Your code compute the <img src="http://chart.googleapis.com/chart?cht=tx&chl=\sum_{i=1}^N alpha[i] * y[i] * K[i, j]" style="border:none;"> by using the `SVMPredictor` which you set bias to 0. In this way, you compute the term AND use a sign function. But in the SVM algorithm, the sign function is not used in computing the bias term.

**Comment by MatrixRanger98** [comment]
Yes. We shouldn't use sign function here. That is a major mistake rather than a bug. I highly suggest correcting it as there are already many webpages on svm quoting this repo.
